process/block/baseProcess.go

```diff
@@ -128,6 +133,108 @@ type bootStorerDataArgs struct {
 	epochStartTriggerConfigKey []byte
 }
 
+func NewBaseProcessor(arguments ArgBaseProcessor) (*baseProcessor, error) {
+	err := checkProcessorParameters(arguments)
+	if err != nil {
+		return nil, err
+	}
+
+	processDebugger, err := createDisabledProcessDebugger()
+	if err != nil {
+		return nil, err
+	}
+
+	mbSelectionSession, err := NewMiniBlocksSelectionSession(
+		arguments.BootstrapComponents.ShardCoordinator().SelfId(),
+		arguments.CoreComponents.InternalMarshalizer(),
+		arguments.CoreComponents.Hasher(),
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	executionResultsTracker := executionTrack.NewExecutionResultsTracker()
+	err = process.SetBaseExecutionResult(executionResultsTracker, arguments.DataComponents.Blockchain())
+	if err != nil {
+		return nil, err
+	}
+
+	execResultsVerifier, err := NewExecutionResultsVerifier(arguments.DataComponents.Blockchain(), executionResultsTracker)
```

Comment:
I hereby confirm that it's perfectly fine to instantiate this light & easy to construct component here (no need to be injected as constructor dependency). Additionally, mocking this specific component, for testing, does not add extra value.

:+1: